api/envoy/extensions/transport_sockets/tls/v3/tls.proto

```diff
@@ -53,6 +54,11 @@ message UpstreamTlsContext {
   //
   // Defaults to 1, setting this to 0 disables session resumption.
   google.protobuf.UInt32Value max_session_keys = 4;
+
+  // Currently, the default value is false, but it is expected to be changed to true by default in a future release.
```

Comment:
Please document what this setting does, and what conditions result in invalid rsa key usage.